src/conf_mode/tftp_server.py

```diff
@@ -69,6 +70,7 @@ def verify(tftpd):
         if not is_addr_assigned(address):
             print(f'WARNING: TFTP server listen address "{address}" not ' \
                   'assigned to any interface!')
+        verify_vrf(tftpd['listen_address'][address])
```

Comment:
Please change iterator to iterate over the dictionary using key/value pairs - `for address, address_config in tftp['listen_address'].items()` to make reading the code easier and not iterate over multiple dictionaries.

Reply:
Done